src/Core/Common/AddNoiseToBinaryImage/Code.cxx

```diff
@@ -63,10 +59,7 @@ main(int argc, char * argv[])
   }
 
   // Write the file
```

Comment:
As this comment now applies to only one line, and that line invokes `WriteImage`, the comment is now useless. Removal is preferred.

Reply:
Done